### ISSUE TITLE
Toasts are not read by screen reader

### ISSUE THREAD

**Issue by danirabbit** [elementary/granite#889]
### What Happened?

When toasts are shown, they aren't read out by the screen reader

### Steps to Reproduce

1. Open Toasts page in Granite Demo
2. Trigger a toast
3. Notice that it's not read by Orca

### Expected Behavior

Toasts should be read out loud when sent

### OS Version

8.x (Circe)

### OS Architecture

amd64 (on most hardwares)

### Session Type

Secure Session (Wayland)

### Software Version

Latest release (I have run all updates)

### Log Output

```shell

```
